sdk/cognitivelanguage/Azure.AI.Language.Conversations/src/ConversationAnalysisOptions.cs

```diff
@@ -22,7 +22,11 @@ public enum ServiceVersion
             /// <summary>
             /// Service version "2021-11-01-preview".
             /// </summary>
-            V2022_03_01_Preview = 1,
+            V2021_11_01_Preview = 1,
+            /// <summary>
+            /// Service version "2022-03-01-preview".
+            /// </summary>
+            V2022_03_01_Preview = 2
```

Comment:
1. Use trailing commas for easier diffing in the future.
2. Our guidelines are that client options are always like `ConversationAnalysisClientOptions`. See https://azure.github.io/azure-sdk/general_design.html.